src/sdl3/pixels.rs

```diff
@@ -5,7 +5,8 @@ use std::convert::{TryFrom, TryInto};
 use std::ffi::c_int;
 use std::fmt::Debug;
 use std::ptr::null;
-use sys::everything::{SDL_PixelFormat, SDL_PixelFormatDetails};
+// Bring in all SDL pixel-related types and constants (for the format enums)
+use sys::everything::*;
```

Comment:
I understand that you're having to access a lot of consts, but can't we avoid including the whole sdl3-sys and maybe include only sys::pixels?